[9.0] [MIG] Backport of slow_statement_logger from 12.0

ThomasBinsfeld opened the pull request:
Backport of #1729

sbidoul committed:
[ADD] slow_statement_logger

ThomasBinsfeld committed:
[MIG] Slow Statement Logger from 12.0 to 9.0

sbidoul commented on slow_statement_logger/__init__.py:
```diff
@@ -7,13 +7,14 @@
 # TODO Odoo option? server_environment?
-LOG_MIN_DURATION_STATEMENT = int(config.get("log_min_duration_statement", "-1"))
+LOG_MIN_DURATION_STATEMENT = int(
+    config.get("log_min_duration_statement", "-1"))
```
@ThomasBinsfeld out of curiosity, is there a reason to wrap this line? Normally MQT accepts 88 characters on all branches now.

ThomasBinsfeld replied:
Oops, i thought it was only on 13.0 branch. Fixed

sbidoul committed:
slow_statement_logger: log with debug level

This is effectively a debugging information and logging
with a warning level would create too much noise in
log capture systems such as Sentry.

sbidoul commented:
@ThomasBinsfeld we need to remove the `static` directory to avoid the module to be autoimported by Odoo. There is also an OCA specific key to add in the manifest to avoid index.html and icon generation (I don't remember it precisely right now, sorry).

ThomasBinsfeld commented:
> @ThomasBinsfeld we need to remove the `static` directory to avoid the module to be autoimported by Odoo. There is also an OCA specific key to add in the manifest to avoid index.html and icon generation (I don't remember it precisely right now, sorry).

Done. The key in the manifest is `"preloadable": False,`.

sbidoul reviewed (approved):
Thanks! The travis error seems unrelated and I suppose nobody is interested to make this old branch green again.

houssine78 reviewed (approved):
LGTM

remytms reviewed (approved):
Seams OK. :)

dreispt commented:
Build is red, please fix
